packages/contracts/contracts/implementations/SdeBitmapsCore.sol

```diff
@@ -3,131 +3,134 @@ pragma solidity 0.8.18;
+  }
 
-    /**
-     * @dev Unsets the bit at `index`.
-     * Orignally published in Bitmaps.sol by OpenZeppelin as `unset`
-     */
-    function unsetBit(BitMap storage bitmap, uint256 bitIndex) internal {
-        uint256 bucket = getBucketFromBitIndex(bitIndex);
-        uint256 mask = 1 << (bitIndex & 0xff);
-        bitmap._data[bucket] &= ~mask;
-    }
+  /**
+   * @dev Unsets the bit at `index`.
+   * Orignally published in Bitmaps.sol by OpenZeppelin as `unset`
+   */
+  function unsetBit(BitMap storage bitmap, uint256 bitIndex) internal {
+    uint256 bucket = getBucketFromBitIndex(bitIndex);
+    uint256 mask = 1 << (bitIndex & 0xff);
+    bitmap._data[bucket] &= ~mask;
+  }
 
-    /**
-     * @dev Sets the bit indices in `indeices` in the bucket at `bucketIndex`.
-     */
-    function setBitsInBucket(
-        BitMap storage _bitmap,
-        uint256 bucketIndex,
-        uint8[] calldata indeices
-    ) internal {
-        uint256 bucket = _bitmap._data[bucketIndex];
-        for (uint256 i = 0; i < indeices.length; i++) {
-            bucket |= 1 << indeices[i];
-        }
-        _bitmap._data[bucketIndex] = bucket;
+  // TODO: tests
+  /**
+   * @dev Sets the bit indices in `indeices` in the bucket at `bucketIndex`.
+   */
+  function setBitsInBucket(
+    BitMap storage _bitmap,
+    uint256 bucketIndex,
+    uint8[] calldata indeices
+  ) internal {
+    uint256 bucket = _bitmap._data[bucketIndex];
+    for (uint256 i = 0; i < indeices.length; i++) {
+      bucket |= 1 << indeices[i];
     }
+    _bitmap._data[bucketIndex] = bucket;
+  }
 
-    /**
-     *
-     * @dev Unsets the bit indices in `indeices` in the bucket at `bucketIndex`.
-     */
-    function unsetBitsInBucket(
-        BitMap storage _bitmap,
-        uint256 bucketIndex,
-        uint8[] calldata indeices
-    ) internal {
-        uint256 bucket = _bitmap._data[bucketIndex];
-        for (uint256 i = 0; i < indeices.length; i++) {
-            bucket &= ~(1 << indeices[i]);
-        }
-        _bitmap._data[bucketIndex] = bucket;
+  // TODO: tests
+  /**
+   *
+   * @dev Unsets the bit indices in `indeices` in the bucket at `bucketIndex`.
+   */
+  function unsetBitsInBucket(
+    BitMap storage _bitmap,
+    uint256 bucketIndex,
+    uint8[] calldata indeices
+  ) internal {
+    uint256 bucket = _bitmap._data[bucketIndex];
+    for (uint256 i = 0; i < indeices.length; i++) {
+      bucket &= ~(1 << indeices[i]);
     }
+    _bitmap._data[bucketIndex] = bucket;
+  }
 
-    /**
-     * @dev Returns bucket at `bucketIndex`, masked to only include bits between `_start` and `_end`.
-     */
-    function selectInsideRange(
-        BitMap storage _bitmap,
-        uint256 _bucketIndex,
-        uint8 _start,
-        uint8 _end
-    ) internal view returns (uint256 bits) {
-        uint256 mask = (type(uint256).max << _start) &
-            (type(uint256).max >> (255 - _end));
-        bits = getBucket(_bitmap, _bucketIndex) & mask;
-    }
+  // TODO: tests
```

Comment:
Premium documentation lol

Reply:
You could also use github issues instead of this

Reply:
That's probably the intelligent solution here hahaha.